src/vs/platform/files/common/fileService.ts

```diff
@@ -901,19 +901,19 @@ export class FileService extends Disposable implements IFileService {
 
 	watch(resource: URI, options: IWatchOptions = { recursive: false, excludes: [] }): IDisposable {
 		let watchDisposed = false;
-		let watchDisposable = toDisposable(() => watchDisposed = true);
+		let disposeWatch = () => { watchDisposed = true; };
```

Comment:
This is just a :lipstick: change right?

Reply:
Yes, it is a cosmetic change to avoid a false positive with the Disposable tracking that I added. The initial version was creating a Disposable via `toDisposable` which was setting the flag to true. But that particular initial disposable was never disposed in the happy case, since it got overwritten with the real disposable. The tool I wrote to detect Disposable leaks would always flag this particular code location. It is technically accurate, that the Disposable was "leaking", but this was not a real Disposable, it was just a code pattern kind of Disposable. So long story short, I just switched to use a simple method.